Api.Flickr/Api.Flickr.csproj

```diff
@@ -5,4 +5,10 @@
         <LangVersion>7.3</LangVersion>
     </PropertyGroup>
 
+    <ItemGroup>
+      <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
+      <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
+      <PackageReference Include="System.Dynamic.Runtime" Version="4.3.0" />
+    </ItemGroup>
```

Comment:
Both libraries have the same dependencies. Is it a good idea to create a separate library project and move their dependencies to single project?

Reply:
Better to make DI in current project.

Reply:
What do you mean? In which projects?